src/models/__tests__/token.test.integration.ts

```diff
@@ -0,0 +1,259 @@
+import * as Knex from 'knex';
+import * as moment from 'moment';
+import { getConfig } from '../../../config';
+import { getTrx } from '../../../tests/utils/database';
+import { Users } from '..';
+import { Tokens } from '../token';
+
+
+describe('User Model', () => {
+  let trx: Knex.Transaction;
+  const config = getConfig(process.env.NODE_ENV);
+  const knex = Knex(config.knex);
+
+  afterAll(async () => {
+    await knex.destroy();
+  });
+
+  beforeEach(async () => {
+    trx = await getTrx(knex);
+  });
+
+  afterEach(async () => {
+    await trx.rollback();
+  });
+
+  describe('createConfirmAddRoleToken', () => {
```

Comment:
Resolve this comment if all these tests are cut and pasted from the old user model tests without any changes (except for re-naming).